### PR TITLE
Garysegal/switch cli commands

### PR DESCRIPTION
**Pull request by garymsegal-ws**
Update CI-CD examples to use "mend deps" and "mend code".

### CODE REVIEW

File: CI-CD/TeamCity/Mend CLI/Teamcity.sh [hunk 1]
```diff
@@ -31,7 +31,7 @@
 echo "Downloading Mend CLI"
 curl -LJO https://downloads.mend.io/production/unified/latest/linux_amd64/mend && chmod +x mend
 echo "Execute Mend CLI"
-echo "Mend SCA Scan"
-./mend sca -u
-echo "Mend SAST Scan"
-./mend sast
+echo "Mend Cdependencies (SCA) scan"
```
Comment (samq-ws) [inline CI-CD/TeamCity/Mend CLI/Teamcity.sh:34]: typo

File: CI-CD/Bitbucket/Mend CLI/Bitbucket.yaml [hunk 2]
```diff
@@ -38,9 +38,9 @@ pipelines:
...
           echo Downloading Mend CLI
           curl -LJO https://downloads.mend.io/production/unified/latest/linux_amd64/mend && chmod +x mend
-          ### SCA Scan ###
-          echo Start Mend SCA scan
-          ./mend sca -u
-          ### SAST Scan ###
-          echo Start Mend SAST scan
-          ./mend sast
\ No newline at end of file
+          ### dependeincies (SCA) Scan ###
```
Comment (samq-ws) [inline CI-CD/Bitbucket/Mend CLI/Bitbucket.yaml:41]: typo

**Review by samq-ws — CHANGES_REQUESTED** [review]
Please fix the .whitesource file while you are it

File: CI-CD/TeamCity/Mend CLI/Teamcity.sh (hunk 1 above)
Comment (garymsegal-ws) [reply CI-CD/TeamCity/Mend CLI/Teamcity.sh]: Fixed

File: CI-CD/Bitbucket/Mend CLI/Bitbucket.yaml (hunk 2 above)
Comment (garymsegal-ws) [reply CI-CD/Bitbucket/Mend CLI/Bitbucket.yaml]: Fixed

**Review by samq-ws — APPROVED** [review]

**Pull request merged**